DigitalLearningSolutions.Data/DataServices/SupervisorDelegateDataService.cs

```diff
@@ -87,14 +87,16 @@ IEnumerable<string> emails
         }
 
         // TODO: HEEDLS-1014 - Change CandidateID to UserID
-        public void UpdateSupervisorDelegateRecordsCandidateId(IEnumerable<int> supervisorDelegateIds, int delegateUserId)
+        public void UpdateSupervisorDelegateRecordsCandidateId(IEnumerable<int> supervisorDelegateIds, int delegateId)

```

Comment:
This reverts refactored code. As the comment above the line says, we should be using UserId (or delegateUserId) instead of candidateId (or delegateId).